### DISCUSSION TITLE
Can I show GSF's presentations at my sponsor's desk in a conference?

### DISCUSSION THREAD

**Discussion by fullo** [Green-Software-Foundation/discussions#130] [Community Working Group]
Next week, I'll be at a conference regarding [Developer Experience](https://2024.dxday.it) with the double role of co-organizer (with the no-profit association GrUSP.org) and sponsor (with my one-man-band consultancy company). 

I'd love to show the [GSF's projects deck](https://docs.google.com/presentation/d/1avmb1WQZLYmXHD42N3rxoA_3vx2FocIh3igU3HGjDEI/edit#slide=id.g12c457ac437_0_110) and held an AMA about green software practices during coffee breaks. My idea is to let DX practitioner know about, contribute, and add IF to their daily routine and to introduce the idea "Sustainability as Code" as a core principle. 

What should I know about GSF's media usage/distribution? (I will not add my logo on those, change copyrights, etc.)

thanks,
F.

**Comment by russelltrow** [comment]
Hey @fullo great question, we have a very permissive policy, you can find all the details here: https://greensoftware.foundation/policy/trademark

@NAMRATA-WOKE would be able to able any specific questions.

**Reply by NAMRATA-WOKE** [reply]
@fullo you can absolutely use the deck! Please mentioned that some projects are in incubation by the foundation, which means they are very much in the experimental stage and can change significantly in scope and impact. 

We ask that anything you share, you do so with proper attribution to the Green Software Foundation. Given the nature of our work, including the date/version always helps!

**Reply by fullo** [reply]
> you can find all the details here: https://greensoftware.foundation/policy/trademark

thank you @russelltrow I missed that page. 

> you can absolutely use the deck! 

@NAMRATA-WOKE thank you, I will also do it in other conferences in which I'm involved. 

> We ask that anything you share, you do so with proper attribution to the Green Software Foundation. 

as mentioned beforehand I will use the official slides as they are without modification and I will cite the GSF as the author, isn't this the work of a champion/advocate? ;) 

> Given the nature of our work, including the date/version always helps!

good idea, thanks!
